src/Notification/NotificationServer.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace OpenEuropa\EPoetry\Notification;
+
+use OpenEuropa\EPoetry\Server\Server;
+
+class NotificationServer
+{
+    /**
+     * Handler to use.
+     *
+     * @var NotificationHandler
+     */
+    protected $handler;
```

Comment:
This parameter doesn't seem to be of any use since we always end up using the NotificationHandler later.

Reply:
The handler will keep information about the call.
You can see it's use in `tests/Notification/NotificationServerHandlerTest.php` where we use it to run the method `debugLastServerCall`.